packages/datagateway-download/src/downloadConfirmation/dialogContent.component.ts

```diff
@@ -0,0 +1,7 @@
+import { DialogContent as MuiDialogContent, styled } from '@mui/material';
+
+const DialogContent = styled(MuiDialogContent)(({ theme }) => ({
+  padding: theme.spacing(2),
+}));
+
+export default DialogContent;
```

Comment:
Minor comment but does this really need to be in it's own file? I feel as though it's place in `downloadConfirmDialog` was fine as it's such a small component - essentially just applying some styles. `dialogTitle` I understand as it is tested separately and a little bigger...

Reply:
Ah, I see - it's because it's used by both `downloadConfirmDialog` and `downloadRequestResult` - nevermind this comment then!

Reply:
Since I needed it in both `DownloadConfirmDialog` and `DownloadRequestResult`, I thought if I was moving `DialogTitle` to its own file, might as well do the same for `DialogContent`, because otherwise I'd end up with `DialogTitle` in a separate file but `DialogContent` being exported by `downloadConfirmDialog.component`, and it just feels weird to me. What do you think?